input/docs/getting-started/upgrade.md

```diff
@@ -3,6 +3,28 @@ Title: Upgrade instructions
 Description: Upgrade instructions between different versions of Cake
 ---
 
```

Comment:
Add heading: `Cake 1.0 to Cake 1.1`

Reply:
I thought about this, but that means we need to update this heading on every release... Do we want that :thinking:?

Reply:
I gave it a go with the variable version - the downside is that it's not going to generate  stable URL (i.e. changes on every release). Given it's at the beginning of the page, doesn't seem like a big issue

Reply:
Yes, that was the overall idea of the page to have a section with instructions how to update from one release to another. What's the issue with that? I'm not speaking about the page title (which is `Upgrade instructions`). I'm speaking about adding an additional `# Cake 1.0 to Cake 1.1` heading before the content relevant for updating from 1.0 to 1.1.

Reply:
> I gave it a go with the variable version - the downside is that it's not going to generate a stable URL (i.e. changes on every release). Given it's at the beginning of the page, doesn't seem like a big issue

We speak about different things. There should be nothing dynamic on this page. The intent was to have a page with a section containing upgrade instructions for every version.

```
# Cake 1.0 to Cake 1.1

Blablabla

# Cake 0.38.x to Cake 1.0

Blablabla
```

Reply:
The thing is that the upgrade instructions for Cake 1.1.0 to Cake 1.2.0 and 1.3.0 and probably 2.0.0 will be the same: `dotnet tool update Cake.Tool`, etc.

That's why I didn't put a header because this is evergreen until .NET tool is replaced by something else

Reply:
Like it is now doesn't make much sense. `Cake.DotNetTool.Module` is specific to this version and should be below a `Cake 1.0 to Cake 1.1` heading. If we want to have general tool instructions here we should provide them for all supported runners and not only for one (like e.g. done here: https://cakebuild.net/docs/writing-builds/reproducible-builds/pinning-cake-version). But I would suggest to do this in a separate PR to not block the 1.1 blog post

Reply:
:thinking: I'm not sure I agree that `Cake.DotNetTool.Module` (from the point of view of upgrade instructions) is specific to 1.1, because not all users will be upgrading from 1.0 to 1.1... Many will be upgrading from and old 0.38.x, skipping a few 0.38.x versions and 1.0, and going straight into 1.1.

Anyone using the `Cake.DotNetTool.Module` in any old version of Cake, would want to read this call-to-action... To me it's a generic piece of upgrade instructions that should stay there for a while and kept visible when 1.2 comes out, etc.

But anyway, I moved the header down and put the generic instructions above the header, so seems like a good compromise and we can decide what to do in 1.2 or 2.0 whichever comes first.

![image](https://user-images.githubusercontent.com/177608/110222603-cbe09d00-7ea9-11eb-8537-82dcc09051ca.png)